src/results.js

```diff
@@ -31,20 +32,66 @@ class Err {
 	get isErr() { return true }
 	get ok() { return null }
 	get err() { return this._error }
+
 	toString() {
 		if (this._error)
 			return "Error: " + JSON.stringify(this._error)
-		else
-			return "Error"
+		return "Error"
 	}
+
 	toJSON() {
 		let error = (this._error === undefined ? '' : this._error)
 		error = error instanceof Error ? error.toString() : error
 		return { 'Error': error }
 	}
+
+	static _notFound ({ message = 'Not Found', payload, stackTrace } = {}){ 
```

Comment:
why `static` and why `static _notFound` + `_exports.Err.notFound = Err._notFound`?

why not just `notFound() {`

Reply:
> Why static? 

As we want to use this feature like Class.method I'm using the static

> Why _exports.Err.notFound = Err._notFound ?

It's because we exports a function in the Err property
``` js
const _exports = {
	Ok: (value) => new Ok(value),
	Err: (err) => new Err(err)
}
```

So when I import it I can't access the Err's method without activate it like a function `Err().notFound()` , so to avoid this weird usage I needed to create properties inside the export.Err property that receives the err methods

to be honest i didn't likes this implementation but i cant think in other way **_without to broken the current library usage_**